src/allreduce_base.cc

```diff
@@ -39,16 +42,8 @@ AllreduceBase::AllreduceBase(void) {
   err_link = NULL;
   dmlc_role = "worker";
   this->SetParam("rabit_reduce_buffer", "256MB");
-  // setup possible enviroment variable of intrest
-  env_vars.push_back("rabit_task_id");
```

Comment:
Is there any possibility of setting up unit tests in rabit?  Even if you can explain everything to me this time, without tests next time someone (might just be me) will still have no clue on what's the result of changes.

Reply:
added unit tests covering this part.